fatf/tests/test_script.py

```diff
@@ -0,0 +1,15 @@
+def times(x, y):
```

Comment:
Add a top of file descriptor (see google python style guide). Also potentially explain how the functions you want to be tested have to begin with 'test_' in order for pytest to notice them, and the same with file names.